From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kuan-Chih Wang <kuanchihwang@outlook.com>
Date: Sat, 3 Jan 2026 20:33:50 +0800
Subject: [PATCH] Update version to 2.6.8

---
 CMakeLists.txt | 2 +-
 configure.ac   | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8ee643b2..7cb025be 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,7 @@ project (PIO C)
 # The project version number.
 set(VERSION_MAJOR   2   CACHE STRING "Project major version number.")
 set(VERSION_MINOR   6   CACHE STRING "Project minor version number.")
-set(VERSION_PATCH   6   CACHE STRING "Project patch version number.")
+set(VERSION_PATCH   8   CACHE STRING "Project patch version number.")
 mark_as_advanced(VERSION_MAJOR VERSION_MINOR VERSION_PATCH)
 
 # Create version info in autotools parlance for pio_meta.h.
diff --git a/configure.ac b/configure.ac
index 384e9039..d44688da 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 ## Ed Hartnett 8/16/17
 
 # Initialize autoconf and automake.
-AC_INIT(pio, 2.6.6)
+AC_INIT(pio, 2.6.8)
 AC_CONFIG_SRCDIR(src/clib/pio_darray.c)
 AM_INIT_AUTOMAKE([foreign serial-tests])
 
@@ -11,7 +11,7 @@ AM_INIT_AUTOMAKE([foreign serial-tests])
 # AC_DEFINE_UNQUOTED for config.h.
 AC_SUBST([PIO_VERSION_MAJOR]) PIO_VERSION_MAJOR=2
 AC_SUBST([PIO_VERSION_MINOR]) PIO_VERSION_MINOR=6
-AC_SUBST([PIO_VERSION_PATCH]) PIO_VERSION_PATCH=6
+AC_SUBST([PIO_VERSION_PATCH]) PIO_VERSION_PATCH=8
 AC_DEFINE_UNQUOTED([PIO_VERSION_MAJOR], [$PIO_VERSION_MAJOR], [PIO major version])
 AC_DEFINE_UNQUOTED([PIO_VERSION_MINOR], [$PIO_VERSION_MINOR], [PIO minor version])
 AC_DEFINE_UNQUOTED([PIO_VERSION_PATCH], [$PIO_VERSION_PATCH], [PIO patch version])
@@ -19,7 +19,7 @@ AC_DEFINE_UNQUOTED([PIO_VERSION_PATCH], [$PIO_VERSION_PATCH], [PIO patch version
 # Once more for the documentation.
 AC_SUBST([VERSION_MAJOR], [2])
 AC_SUBST([VERSION_MINOR], [6])
-AC_SUBST([VERSION_PATCH], [6])
+AC_SUBST([VERSION_PATCH], [8])
 
 
 # The m4 directory holds macros for autoconf.
-- 
2.47.3

